core/src/test/resources/Http/Responses/specialCharactersInURLParam.txt

```diff
@@ -1,5 +1,5 @@
 HOST 127.0.0.1
-GET /value%20value?param=Hello+G%C3%BCnter&otherKey=otherValue HTTP/1.1
```

Comment:
Previously the `%20` from the test was decoded into a space and re-encoded as `+`, now because of the changes that is not done.
Since both can be used in the URL query part this should be inconsequential.